sources/assets/Stride.Core.Assets/StridePackagesToUpgrade.cs

```diff
@@ -0,0 +1,79 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Stride.Core.Assets;
+public static class StridePackagesToUpgrade
```

Comment:
Nit-pick: separate by an empty line.

```suggestion
namespace Stride.Core.Assets;

public static class StridePackagesToUpgrade
```